dashboard/app/views/projects/index.html.haml

```diff
@@ -69,6 +69,10 @@
       $scope.projectsLoaded = true;
     }).catch($scope.genericError);
 
+    $scope.projectVisible = function(project) {
+      return (!project.hidden);
```

Comment:
Ok... how do we know what to fetch? Do we need another api to store a list
of projects?

On Fri, Feb 20, 2015 at 1:26 PM, Josh Lory <notifications@github.com> wrote:

> In dashboard/app/views/projects/index.html.haml
> <https://github.com/code-dot-org/code-dot-org/pull/1360#discussion_r25104443>
> :
>
> > @@ -69,6 +69,10 @@
> >        $scope.projectsLoaded = true;
> >      }).catch($scope.genericError);
> >
> > +    $scope.projectVisible = function(project) {
> > +      return (!project.hidden);
>
> I could see an API where fetches for the hidden projects would still
> succeed, they just wouldn't be enumerated when querying.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/1360/files#r25104443>.
>